modules/functions/src/controllers/utils.ts

```diff
@@ -0,0 +1,127 @@
+import ConduitGrpcSdk, {
+  ConduitRouteActions,
+  GrpcError,
+  ParsedRouterRequest,
+  RouteBuilder,
+} from '@conduitplatform/grpc-sdk';
+import { FunctionDefinitions, Functions } from '../models';
+import { isNil } from 'lodash';
+import { NodeVM } from 'vm2';
+import { status } from '@grpc/grpc-js';
+
+function getOperation(op: string) {
+  switch (op) {
+    case 'GET':
+      return ConduitRouteActions.GET;
+    case 'POST':
+      return ConduitRouteActions.POST;
+    case 'UPDATE':
+      return ConduitRouteActions.UPDATE;
+    case 'DELETE':
+      return ConduitRouteActions.DELETE;
+    case 'PATCH':
+      return ConduitRouteActions.PATCH;
+    default:
+      return ConduitRouteActions.GET;
+  }
+}
+
+async function executeFunction(
+  call: ParsedRouterRequest,
+  callback: any,
+  functionCode: string,
+  timeout: number,
+  name: string,
+  grpcSdk: ConduitGrpcSdk,
+) {
+  const logs: string[] | undefined = [];
+  const vm = new NodeVM({
+    console: 'redirect',
+    sandbox: {},
+    timeout: timeout,
+    require: {
+      external: true,
+      import: ['lodash'],
```

Comment:
make sure that lodash is in package.json, and also add here and in package.json "axios".